src/scripts/no_recommended/hide_blog_carousels.js

```diff
@@ -7,7 +7,7 @@ const hiddenClass = 'xkit-no-recommended-blog-carousels-hidden';
 const styleElement = buildStyle(`
   .${hiddenClass} { position: relative; }
   .${hiddenClass} > div { visibility: hidden; position: absolute; max-width: 100%; }
-  .${hiddenClass} > div img, .${hiddenClass} > div canvas { visibility: hidden; }
+  .${hiddenClass} > div ${keyToCss('visible', 'poster')}, .${hiddenClass} > div canvas { display: none }
```

Comment:
1. Is there a purpose to using the CSS map here other than future-proofing? I can't imagine Tumblr would start using a third image-holding element... if there were any advantages to `<picture>` I'm sure it would be being used already
2. If we were to go back to just targeting element types, we could simplify this line down very nicely

```suggestion
  .${hiddenClass} > div :is(img, video, canvas) { display: none }
```

Reply:
Nope, not particularly! I think my line of logic was just "target the classes that redpop would be setting `visibility` on and the elements xkit would be setting `visibility` on."

Reply:
Solid process, unfortunately "visible" by itself is not too descriptive. Ah the joys of dealing with the result of SCSS without seeing the source sheets... in the same context, anyway.